src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -1103,41 +1106,6 @@ namespace BaseboardRadiator {
         state.dataLoopNodes->Node(WaterOutletNode).Enthalpy = baseboard->baseboards(BaseboardNum).WaterOutletEnthalpy;
     }
 
-    Real64 HWBaseboardUAResidual(EnergyPlusData &state,
```

Comment:
Most residual functions can be deleted entirely.  Since we access the captured parameters directly by name, we don't need to have an extra set of documentation to coerce the variables back out of the Par array.